docs/modules/ROOT/pages/access.adoc

```diff
@@ -1,672 +1,414 @@
-:ownable-cairo: link:https://github.com/OpenZeppelin/cairo-contracts/blob/release-v0.6.1/src/openzeppelin/access/ownable/library.cairo[Ownable]
-
+:ownable-cairo: link:https://github.com/OpenZeppelin/cairo-contracts/blob/release-0.7.0/src/access/ownable/ownable.cairo[Ownable]
+:sn_keccak: https://docs.starknet.io/documentation/architecture_and_concepts/Cryptography/hash-functions/#starknet_keccak[sn_keccak]
 :extensibility-pattern: xref:extensibility.adoc#the_pattern
 
 = Access
 
-CAUTION: Expect these modules to evolve.
-
 Access control--that is, "who is allowed to do this thing"--is incredibly important in the world of smart contracts.
```

Comment:
```suggestion
Access control--that is, "who is allowed to do this thing"—is incredibly important in the world of smart contracts.
```
It's displayed as two dashes instead of an em dash